### DISCUSSION TITLE
功能支持

### DISCUSSION THREAD

**Discussion by canxin121** [canxin121/app_rhyme#10] [Ideas]
Todo：
- 更多音乐平台数据支持
- 歌曲元数据匹配/写入
- ~~歌单导入~~/订阅
- ~~自定义数据路径~~
- ~~更多可交互ui~~
- ~~应用本体更新提示~~
- ~~第三方音乐源通过链接导入并自动更新~~
- 搜索某歌手的歌曲(制成歌单形式)
- 搜索某歌手的专辑(不一定能实现)
- 搜索历史
- 歌曲推荐
- 音乐排行榜
- ~~黑夜模式~~
- i18n
- 定时关闭
- 歌单云同步
- 悬浮歌词
- 锁屏/车机歌词(副标题用作歌词显示)
- Carplay车机操作界面
- ~~侧滑返回~~

**Comment by Vincentvip** [comment]
app非常好，希望加入暗黑模式。谢谢！

**Comment by ShopAI8** [comment]
怎么载入音乐源捏

**Reply by canxin121** [reply]
目前只能下载evc文件之后导入，可以自行制作evc文件或者issue置顶看看有无别人的，下载之后设置里面导入即可。

**Comment by fdgdssds** [comment]
多久可以更新

**Reply by canxin121** [reply]
下周二期末周结束后，就有时间了。

**Reply by fdgdssds** [reply]
好的，现在巨魔用户挺多，iOS需求性还是挺多的。

**Comment by maomaovip** [comment]
iphone 15plus 
ios17.1.2平台最新版的打开是白屏

**Reply by canxin121** [reply]
> iphone 15plus 
> ios17.1.2平台最新版的打开是白屏

阿哲，试下Action中的包，这种问题很难debug，因为我无法得知问题原因，也没有mac开发环境

**Comment by maomaovip** [comment]
好的，谢谢，我刚刚把旧的删除重新装新的，打开就正常了

发送自 Windows 10 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>应用

发件人: ***@***.***>
发送时间: 2024年7月14日 0:47
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [canxin121/app_rhyme] 功能支持 (Discussion #10)


iphone 15plus
ios17.1.2平台最新版的打开是白屏

阿哲，试下Action中的包，这种问题很难debug，因为我无法得知问题原因，也没有mac开发环境

―

**Comment by Subei2023** [comment]
希望增加锁屏歌词

**Comment by ROC4FUN** [comment]
ios14.3 打开后过几秒闪退，求解决

**Comment by Alan-in** [comment]
如果可以在下载歌曲的时候把歌词一起下载下来就好了

**Comment by DDDDADDDD** [comment]
怎么导入本地音乐？

**Comment by 1GuAna** [comment]
请问歌词默认下载位置是哪呢？Documents里只有MP4文件

**Comment by IggyMeraki** [comment]
大佬可以做一个歌单导出吗

**Comment by p0erish1** [comment]
现在是只能导入wy一个平台的歌单吗，求大佬做一下其他平台的

**Comment by spikesgithub** [comment]
请问现在有的歌只能听30s吗

**Comment by 3thousandr** [comment]
请问，能否添加蓝牙歌词功能？

**Comment by mzz2345gj** [comment]
增加英文语言支持

**Comment by BreadKat0707** [comment]
目前我正在使用Action构建的2.0.1版本，遇到不可以创建/导入歌单的问题
[error] | 0:07:59 582ms | [selectPlaylistCollectionDialog] Failed to get playlist collections from db: AnyhowException(Database is not inited.

Stack backtrace:
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: _ZL15__pthread_startPv
  10: __start_thread)
然后现在的缓存路径选择还是有点不是很好理解，希望以后逻辑会更明晰一些
平板模式下貌似没办法调整播放进度(？)

**Reply by canxin121** [reply]
> 目前我正在使用Action构建的2.0.1版本，遇到不可以创建/导入歌单的问题
> 
> [error] | 0:07:59 582ms | [selectPlaylistCollectionDialog] Failed to get playlist collections from db: AnyhowException(Database is not inited.
> 
> 
> 
> Stack backtrace:
> 
>    0: <unknown>
> 
>    1: <unknown>
> 
>    2: <unknown>
> 
>    3: <unknown>
> 
>    4: <unknown>
> 
>    5: <unknown>
> 
>    6: <unknown>
> 
>    7: <unknown>
> 
>    8: <unknown>
> 
>    9: _ZL15__pthread_startPv
> 
>   10: __start_thread)
> 
> 然后现在的缓存路径选择还是有点不是很好理解，希望以后逻辑会更明晰一些
> 
> 平板模式下貌似没办法调整播放进度(？)
我不好说，可以尝试删了重下

**Reply by canxin121** [reply]
> 目前我正在使用Action构建的2.0.1版本，遇到不可以创建/导入歌单的问题
> 
> [error] | 0:07:59 582ms | [selectPlaylistCollectionDialog] Failed to get playlist collections from db: AnyhowException(Database is not inited.
> 
> 
> 
> Stack backtrace:
> 
>    0: <unknown>
> 
>    1: <unknown>
> 
>    2: <unknown>
> 
>    3: <unknown>
> 
>    4: <unknown>
> 
>    5: <unknown>
> 
>    6: <unknown>
> 
>    7: <unknown>
> 
>    8: <unknown>
> 
>    9: _ZL15__pthread_startPv
> 
>   10: __start_thread)
> 
> 然后现在的缓存路径选择还是有点不是很好理解，希望以后逻辑会更明晰一些
> 
> 平板模式下貌似没办法调整播放进度(？)  

我不好说，可以尝试删了重下

**Comment by montau59** [comment]
hello，安卓14, 版本2.0.1, 播放页上滑特别卡顿

**Comment by AnserBaby** [comment]
请问有没有支持Harmony鸿蒙的计划？

**Comment by jojhken** [comment]
js音源改格式为evc能行吗
